janusgraph-all/pom.xml

```diff
@@ -49,11 +49,6 @@
             <version>${hbase098.core.version}-hadoop2</version>
         </dependency>
         <dependency>
-            <groupId>${project.groupId}</groupId>
-            <artifactId>janusgraph-hadoop</artifactId>
```

Comment:
You're quite right.  This dependency is used to output janusgraph-hadoop-0.1-SNAPSHOT.jar in the final lib folder and without it the jar is dropped, my bad.   

The issue is that this is a dependency on a module that doesn't have any classes, it's just an assembly project.  So it's not a classpath dependency, it's a packaging dependency.  I'm not sure how to handle this nicely in Maven, suggestions are welcome.  I've added the dependency back int.